ui/raidboss/data/06-ew/raid/p1s.ts

```diff
@@ -0,0 +1,206 @@
+    },
+    {
+      id: 'P1S Slam Shut',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6617', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6617', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6617', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6617', source: 'エリクトニオス', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P1S Gaoler\'s Flail Right => Left',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '65F6', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '65F6', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '65F6', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '65F6', source: 'エリクトニオス', capture: false }),
+      alertText: (_data, _matches, output) => output.combo!({ first: output.r!(), second: output.l!() }),
+      outputStrings: flailDirections,
+    },
+    {
+      id: 'P1S Gaoler\'s Flail Left => Right',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '65F7', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '65F7', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '65F7', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '65F7', source: 'エリクトニオス', capture: false }),
+      alertText: (_data, _matches, output) => output.combo!({ first: output.l!(), second: output.r!() }),
+      outputStrings: flailDirections,
+    },
+    {
+      id: 'P1S Gaoler\'s Flail Out => In',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '65F8', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '65F8', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '65F8', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '65F8', source: 'エリクトニオス', capture: false }),
+      alertText: (_data, _matches, output) => output.outThenIn!(),
+      outputStrings: {
+        outThenIn: Outputs.outThenIn,
+      },
+    },
+    {
+      id: 'P1S Gaoler\'s Flail In => Out',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '65F9', source: 'Erichthonios', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '65F9', source: 'Erichthonios', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '65F9', source: 'Érichthonios', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '65F9', source: 'エリクトニオス', capture: false }),
+      alertText: (_data, _matches, output) => output.inThenOut!(),
+      outputStrings: {
+        inThenOut: Outputs.inThenOut,
+      },
```

Comment:
These have multiple spell ids dependent on which is first.

Should be for Out => In:
['65F8', '65F9']

and for In => Out:
['65FA', '65FB']